### DISCUSSION TITLE
How to use the built-in components in rspress

### DISCUSSION THREAD

**Discussion by asurance** [web-infra-dev/rspress#1097] [Q&A]
[内置组件](https://rspress.dev/zh/guide/default-theme/components)我看这边的组件都是从`@themes`中导出的，这个是啥，是要再安装一个包吗，另外，我是在`modern.js module`中使用的，使用的pnpm安装的`@modern-js/plugin-rspress`，这个为啥会提示没有对应的类型到处，pnpm版本`8.15.8`

**Comment by Timeless0911** [comment]
`@theme` is alias which is dealing during the rspress build process.
https://github.com/web-infra-dev/rspress/blob/51f150aa8bea22dca3705f71db53d85e64741f6b/packages/core/src/node/initRsbuild.ts#L146

**Reply by asurance** [reply]
所以这边的@theme是可以直接使用的是吧

**Reply by Timeless0911** [reply]
yep

**Comment by Timeless0911** [comment]
The type issue you can see a demo in community https://github.com/erhulee/webcare for reference.

**Reply by asurance** [reply]
这边能具体到文件吗，我不清楚是需要参考哪些地方的配置来解决的类型问题

**Reply by Timeless0911** [reply]
应该不会有类型问题，有的话你应该检查一下自己的 `tsconfig.json`，这个项目你可以去看看他是否有你说的问题，如果有，你可以提一个 issue 并给出复现 demo

**Reply by Timeless0911** [reply]
你可以参考一下这个 pr 的改动，把你的 `tsconfig` 梳理一下，module 构建的时候使用一个，根目录对`modern.config.ts` 配置文件也可以有一个 https://github.com/web-infra-dev/rspress/pull/1101/files
